pkg/activator/dedupe.go

```diff
@@ -17,15 +17,16 @@ limitations under the License.
 package activator
 
 import (
-	"fmt"
 	"net/http"
 	"sync"
+
+	"github.com/pkg/errors"
```

Comment:
Can this be just "errors"? Any particular preference using pkg/errors over errors?

Reply:
Ditto here. Automation fail.